Dockerfile

```diff
@@ -14,3 +14,13 @@ RUN pip install ansible
 # Install Ansible inventory file.
 RUN mkdir -p /etc/ansible; \
 echo -e '[local]\nlocalhost ansible_connection=local' > /etc/ansible/hosts
+ENV ANSIBLE_USER=ansible
+ENV SUDO_GROUP=wheel
+RUN set -xe \
+  && groupadd -r ${ANSIBLE_USER} \
+  && useradd -m -g ${ANSIBLE_USER} ${ANSIBLE_USER} \
+  && usermod -aG ${SUDO_GROUP} ${ANSIBLE_USER} \
+  && sed -i "/^%${SUDO_GROUP}/s/ALL\$/NOPASSWD:ALL/g" /etc/sudoers \
```

Comment:
Having the NOPASSWD on the ansible user seems necessary when installing base-devel, any needed dependencies and finally the built package, but it would be better to restrict the policy to only that user, or maybe to a ci group and assign the user to that group. What do you think?

Reply:
I thought about both creating policy directly for ansible user and about creating a dedicated group but decided to go with the wheel because it is just simpler and it is easier to read because wheel group is well know. Another consideration is that having "ansible" user as part of "wheel" group is not less secure than running everything from under "root" (as it is the case right now).  We can make a dedicated "ci" group if you insist, I do not have a super-strong opinion about that, but my personal preference is to re-use wheel group for the reasons above.

Reply:
You are correct, it's not strictly a matter of security, since everything runs as root right now, but more of expectations: a pristine Arch installation would have no user and execute everything under root. This is expected. The wheel group, as you pointed out, is the de-facto group for sudoers on arch, another expected fact. This image is a not-really-pristine arch dedicated to Ansible, so an ansible user with NOPASSWD makes sense and could be expected. What's not going to be expected is that, by default, **every** members of wheel can elevate without password.
I'd rather have a dedicated and documented policy, either for the ansible user, it's ansible default group, or a dedicated ci group.

At the end of the day, this is something that only directly impacts this image, in the sense that the workflow in any derivate image would not change (as in, the ansible user would still be able to elevate without password anyway).

Reply:
Done, could you please check?